src/task/ReceiveRemoteVideoPauseResumeTask.ts

```diff
@@ -36,6 +37,32 @@ export default class ReceiveRemoteVideoPauseResume
     this.context.removableObservers.push(this);
   }
 
+  updateSubscribedGroupdIds(groupIds: Set<number>): void {
+    const existingVideoTileIds = new Set<number>();
+    for (const groupId of groupIds) {
+      const attendeeId = this.context.videoStreamIndex.attendeeIdForGroupId(groupId);
+      if (attendeeId.length === 0) {
+        this.logger.warn(`Could not find attendee ID for newly subscribed group ID ${groupId}`);
+        continue;
+      }
+      let videoTile = this.context.videoTileController.getVideoTileForAttendeeId(attendeeId);
+      if (videoTile === undefined) {
+        this.logger.info(
+          `No existing video tile for attendee ID ${attendeeId} with new group ID ${groupId}. Creating new one.`
+        );
+        videoTile = this.context.videoTileController.addVideoTile();
+        videoTile.bindVideoStream(attendeeId, false, null, 0, 0, 0, null);
+      }
+      existingVideoTileIds.add(videoTile.id());
```

Comment:
A verifying check: This should result in `videoTileDidUpdate` and the builder app can accordingly handle this right?

Reply:
Yes, but there will not be any video stream bound to it until the negotiation completes.